0xCristianOff/GCASH-project
# Add founder rewards

* Adds founder rewards. A configurable percentage of each block reward goes to the specified address.
* Currently scheduled to take effect at a height of 14000.

Testing instructions:

* You may wish to test this out on a testnet, so you don't have to wait until block 14000. I used this guide to setup a local testnet: https://docs.turtlecoin.lol/developer/local-testnet
* If using a local testnet, you probably want to change the height the founder rewards are activated to zero, and replace the address+private view key with one you have created.
* Then, simply start mining, to a *different* address than the one the founder rewards go to.
* Monitor the balance of the two address.
* The one you are mining to should have received 95% of the funds mined.
* The founder wallet should have 5% of the funds mined.
* Try using an unupdated daemon+miner combo - or change the founder reward height to something higher than the current height, and start mining.
* The updated daemon should reject any blocks found, with a warning that the founder reward is not present.
